### ISSUE TITLE
As a user I would like each biotope page to link to the relevant page on marlin.ac.uk

### ISSUE THREAD

**Issue by CathyJinJNCC** [jncc/marine-habitat-classification#27]
Looks like they only have pages for the lowest levels of the classification so would not be able to put a link on each page, also should store this in the db

**Comment by helen-lillis** [comment]
Yes that's right, they don't have a page for every biotope. I will request a look-up table from Marlin that gives the URL for each biotope code. Then I suppose the info can be added to the WEB_BIOTOPE table in the db. The link would be included on the biotope description page under a section entitled "Sensitivity Assessment", with the text "A comprehensive sensitivity assessment for this biotope, can be found on the *Marlin website* <hyperlink>. For more information about the method, see *How is sensitivity assessed* <https://www.marlin.ac.uk/sensitivity/sensitivity_rationale>".

**Comment by helen-lillis** [comment]
Update: I've got the look-up table. It's here:
Z:\Marine\Evidence\HabitatClassification\JNCC_MarineClassification\Development\NewBiotopeProject\Publication\Website\LinksToSensitivityAssessments.csv
